client/components/PubEdgeListing/pubEdgeListingCard.scss

```diff
@@ -5,18 +5,25 @@
 	border-width: 1px;
 	margin-bottom: 24px;
 	position: relative;
-	transition: filter 100ms linear;
+	transition: all 100ms linear;
```

Comment:
Changed to `all` to transition border color from gray to community accent color when it is set.